pkg/cinderapi/volumes.go

```diff
@@ -23,22 +25,22 @@ func GetVolumes(parentName string, name string) []corev1.Volume {
 		},
 	}
 
-	return append(cinder.GetVolumes(parentName, false), backupVolumes...)
+	return append(cinder.GetVolumes(parentName, false, extraVol, []storage.PropagationType{cinder.Cinder, cinder.CinderAPI}), backupVolumes...)
```

Comment:
Shouldn't `[]storage.PropagationType{cinder.Cinder, cinder.CinderAPI}` be a constant we reuse all over this file?

Reply:
Done